src/pkg/manifests/developer:opensolaris:pkg5.p5m

```diff
@@ -27,13 +27,12 @@ set name=pkg.description \
     value="This package should be installed on systems used for development of the pkg(5) project"
 set name=info.classification \
     value="org.opensolaris.category.2008:Development/Distribution Tools"
-depend type=conditional fmri=web/server/apache-24/module/apache-wsgi-39 \
```

Comment:
A bit of history here:

Previously there was a `require-any` dependency and two `conditional` dependencies (one for Python 3.5 and the other one for Python 3.9).  This was okay and worked as designed and expected.  Yes, for that case those predicates had to be as they were.  Such setup just makes sure that there is `mod_wsgi` installed for both Python versions (3.5 and 3.9) if such Pythons are installed.  I assume this was needed for smooth transition of `pkg` (including the ips server) from 3.5 to 3.9.

When I dropped dependency on Python 3.5 via #116 back in 2022 I removed one conditional dependency (for Python 3.5) and left the other one (for Python 3.9).  I also converted the `require-any` to `require`.  This change was technically incorrect (because one dependency - the `conditional` one - was left as a duplicate) but otherwise harmless, because `pkg` deals with that properly.

This change here just fixes the minor lapse I did previously.

Thank you.

Reply:
Oh, thats also already 2 years ago. Thanks for the Update. LGTM from me.